src/teleop/mrover_arm/.setup_assistant

```diff
@@ -8,4 +8,4 @@ moveit_setup_assistant_config:
   CONFIG:
     author_name: Joe Nanni
     author_email: jnnanni@umich.edu
-    generated_timestamp: 1676059263
```

Comment:
Any idea why these changes might have popped up?

Reply:
Probably because I pulled an older version of this file? I never opened the setup assistant on this branch.

Reply:
Some programs automatically add new lines to the end of files. If this got touched at all it could have been changed but not a big deal